### PR TITLE
created github actions

### PR DESCRIPTION
**Pull request by shamis**
replace travis with github actions
to be merged after #11

### CODE REVIEW

File: .github/workflows/erlang.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    container:
+      image: erlang:22.0.7
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Compile
+      run: rebar3 compile
+    - name: Check formatting
+      run: DEBUG=1 rebar3 steamroll --check
```
Comment (c-bik) [inline .github/workflows/erlang.yml:23]: Why DEBUG=1 still?

File: .travis.yml [hunk 2]
```diff
@@ -3,10 +3,10 @@ install:
   - wget https://s3.amazonaws.com/rebar3/rebar3 && chmod +x rebar3
 language: erlang
 otp_release:
-  - 20.1
-  - 20.0
-  - 19.3
+  - 22.2
+  - 21.3
```
Comment (c-bik) [inline .travis.yml:7]: This file should be deleted!

**Review by c-bik — CHANGES_REQUESTED** [review]
Clean ups.

File: .github/workflows/erlang.yml (hunk 1 above)
Comment (shamis) [reply .github/workflows/erlang.yml:23]: strangely format check is failing on github actions and working locally and on travis. I was checking if I could get more information, but was not helpful

File: .travis.yml (hunk 2 above)
Comment (shamis) [reply .travis.yml:7]: yes, have to be deleted once the actions are finalised. I could not make the actions run without a PR

**Review by c-bik — APPROVED** [review]

**Pull request merged**